simphony/testing/abc_check_mesh.py

```diff
@@ -384,11 +400,14 @@ def test_update_item_points(self):
         container = self.container
         uids = self._add_items(container)
         item = self.get_operation(container, uids[2])
+        point_uids = [
+            container.add_point(Point((1.0 * i, 1.0 * i, 1.0 * i)))
```

Comment:
The points added at the setup are used in all the tests which relay on a specific number of points available. However, the points added here make sense only for the this test. It was easier to add a bunch of points here that are going to be used and update the `points` attribute of a Mesh element.